sdk/src/main/java/com/silanis/esl/sdk/internal/converter/AccountRoleConverter.java

```diff
@@ -0,0 +1,37 @@
+package com.silanis.esl.sdk.internal.converter;
+
+import com.google.common.base.Optional;
+import com.silanis.esl.sdk.AccountRole;
+
+
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+public class AccountRoleConverter {
+    private final Optional<AccountRole> sdkAccountRoleOptional;
+    private final Optional<com.silanis.esl.api.model.AccountRole> apiAccountRoleOptional;
```

Comment:
Is there any reason to use Optional here?

Reply:
I just followed the pattern on other Converters. Some people find it nicer than to use null check.